js/actions/appActions.js

```diff
@@ -1292,6 +1292,19 @@ const appActions = {
     AppDispatcher.dispatch({
       actionType: appConstants.APP_DRAG_CANCELLED
     })
+  },
+
+  /**
+   * Notifies autoplay has been blocked
+   * @param {string} location - Location of current frame
+   * @param {number} tabId - Tab id of current frame
+   */
+  autoplayBlocked: function (location, tabId) {
+    AppDispatcher.dispatch({
+      actionType: appConstants.APP_AUTOPLAY_BLOCKED,
```

Comment:
Basically it just leaves the chance that someone will use the API wrongly at some point, so better not to include it.